Pod/Classes/Optional Data Sources/VOKFetchedResultsDataSource.m

```diff
@@ -264,31 +264,22 @@ - (BOOL)tableView:(UITableView *)tableView canMoveRowAtIndexPath:(NSIndexPath *)
 
 #pragma mark - Fetched results controller
 
-- (NSFetchedResultsController *)fetchedResultsController
-{
-    if (!_fetchedResultsController) {
-        [self initFetchedResultsController];
```

Comment:
yes, the [init method](https://github.com/chillpop/Vokoder/blob/feature/syntax-formatting/Pod/Classes/Optional%20Data%20Sources/VOKFetchedResultsDataSource.m#L50)